src/main/java/ch/njol/skript/expressions/ExprPlayerlistHeaderFooter.java

```diff
@@ -44,17 +44,17 @@
 @Description("The message above and below the player list in the tab menu.")
 @Examples({"set all players tab list header to \"Welcome to the Server!\"",
 			"send \"%the player's tab list header%\" to player",
-			"reset all player's tab list header"})
+			"reset all players's tab list header"})
 @Since("INSERT VERSION")
-@RequiredPlugins("Spigot 1.13 or newer")
+@RequiredPlugins("1.13 or newer")
 public class ExprPlayerlistHeaderFooter extends PropertyExpression<Player, String> {
 	
 	static {
 		if (Skript.methodExists(Player.class, "setPlayerListHeaderFooter", String.class, String.class)) //This method is only present if the header and footer methods we use are
-			Skript.registerExpression(ExprPlayerlistHeaderFooter.class, String.class, ExpressionType.PROPERTY, "[the] %players% (player|tab) list (1¦header|2¦footer)");
+			Skript.registerExpression(ExprPlayerlistHeaderFooter.class, String.class, ExpressionType.PROPERTY, "[the] %players%['s] (player|tab)[ ]list (header|1¦footer)[s] [(text|message)[s]]");
```

Comment:
Use '[s]; it allows for players' which is grammatically correct.

Reply:
The expression should be converted to SimplePropertyExpression anways